src/Domain/Value/Branch.php

```diff
@@ -154,6 +154,11 @@ public function lowestPhpVersion(): PhpVersion
         return reset($this->phpVersions);
     }
 
+    public function highestPhpVersion(): PhpVersion
+    {
+        return end($this->phpVersions);
```

Comment:
Could you please add a test in `BranchTest` class? Thanks

Reply:
I added the method plus a test in https://github.com/sonata-project/dev-kit/pull/1245